python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -270,6 +271,50 @@ def build(
         )
         return feature_view
 
+    def get_step_size(self) -> timedelta:
```

Comment:
It might be better to declare this method as `@property`. Same for `get_group_by_keys`.